plugins/acpi-wpbt/README.md

```diff
@@ -0,0 +1,15 @@
+ACPI WPBT
+=========
+
+Introduction
+------------
+
+This plugin checks if the WPBT table has been exported. Although unused by
+Linux, Microsoft Windows will unconditionally execute the EXE file specified by
```

Comment:
According to the spec there are conditions to the launch (including failures for lack of signing or system policy).

Reply:
Great, thanks. Added.